Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1288,11 +1286,10 @@ def prepGlyph(g, marker, index=0):
 
 def setMarkerColor(p, marker, c, cmap=None):
     # Color
-    if cmap is None:
-        if marker.colormap is not None:
-            cmap = marker.colormap
-        else:
-            cmap = 'default'
+    if marker.colormap is not None:
```

Comment:
same here. if elif should be used if there is else as well. Unless cmap has some default value.